docs/packaging-an-application/components-and-containers.md

```diff
@@ -193,6 +193,31 @@ Optional properties:
       is_excluded_from_backup: true
 ```
 
+We can use "volumes_from" to attach several mounts from a colocated container.
```

Comment:
Can this become:

"Replicated supports `volumes_from` to attach several mounts from a colocated container."?